docs/persistence.md

```diff
@@ -0,0 +1,110 @@
+# Persistence
+
+Generally speaking, adding database storage or persistence to your Probot App will greatly complicate your life. It's perfectly doable, but for most use-cases you'll be able to manage relevant data within GitHub issues, pull requests and your app's configuration file.
```

Comment:
thoughts on rewording:
`It's perfectly doable, but for most use-cases you'll be able to manage relevant data within GitHub issues, pull requests and your app's configuration file.`  
to =>
 `It's perfectly doable; however, in most use-cases you'll be able to access all the data you need through the GitHub API.`

Reply:
I'm okay with this being a little lengthy because it suggests concrete ways for people to use GitHub's API as storage. It could be something like:

```
It's perfectly doable; however, in most use-cases you'll be able to access all the data you need through the GitHub API by way of issues, pull requests and your app's configuration file.
```